packages/ui-next/package.json

```diff
@@ -71,6 +71,7 @@
   },
   "devDependencies": {
     "@cartridge/eslint": "workspace:^",
+    "@cartridge/presets": "catalog:",
```

Comment:
is this needed?

Reply:
This is used in storybook for mainly layout thus devDependencies.

https://github.com/cartridge-gg/controller/blob/407797509c7c569bbde474676dede08d2b4af4d3/packages/ui-next/.storybook/preview.tsx#L10